shared/test/test_http_cache.rb

```diff
@@ -0,0 +1,349 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
```

Comment:
Could you use require_relative here and below if possible?

The expand_path hack is need in a few places in Sinatra because of the way content is evaluated, but should not be required here.  require_relative is easier to read and doesn't have the confusing extra '..' in the path.